src/components/IndividualArticle.jsx

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from "react";
+import { getArticleById } from "../api";
+import { useParams } from "react-router-dom";
+
+const IndividualArticle = () => {
+    const [currentArticle, setCurrentArticle] = useState('');
+    const { article_id } = useParams();
+
+    useEffect(() => {
+        getArticleById(article_id).then((res) => {
+            res.article.created_at = res.article.created_at.replace('T', ' ').substring(0, 16);
+            setCurrentArticle(res.article);
+        })
+    }, [article_id])
+
```

Comment:
You could add some loading state here to conditionally render something to tell a user the data is still being fetched while they are waiting (for those on a slower internet connection) 💡